AnimationGroup.lua

```diff
@@ -222,8 +239,10 @@ function AG:Fire(group, animation, signal)
     if (signal == 'Finished' and all_finished) or signal == 'Bounce' then
         if group.shifted then
             if not group.finishing then
-                group.reverse = not group.reverse
+                group.reverse = bouncing and (not group.reverse) or false
```

Comment:
```suggestion
                group.reverse = bouncing and (not group.reverse)
```

Both bouncing and (not group.reverse) evaluates to a boolean - no need to fallback false  to false :)